modules/ROOT/pages/dashboard-config-ref.adoc

```diff
@@ -35,69 +35,69 @@ These metrics are available when you use the Basic mode for applications:
+| JVM - Heap Committed | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Heap Used | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - JVM Uptime | `ms` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Memory Utilization | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Metaspace Committed | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Metaspace Init | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Metaspace Max | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Metaspace Used | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par Eden Committed | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par Eden Init | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par Eden Max | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par Eden Used | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par New Collection Count | `short` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par New Collection Time | `ms` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par Survivor Committed | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par Survivor Init | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par Survivor Max | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Par Survivor Used | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Tenured Gen Committed | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Tenured Gen Init | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Tenured Gen Max | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Tenured Gen Used | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Thread count - Server | `short` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Total System Memory | `bytes` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| JVM - Total System Processors | `short` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2, 4.x-AM
+| Outbound - Average Request Count by Endpoint | `short` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Average Request Count by Flow | `short` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Average Response Time by Endpoint | `ms` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Average Response Time by Flow | `ms` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Failure Average Request Count by Endpoint | `short` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Failure Average Request Count by Flow | `short` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Failure Average Response Time by Endpoint | `ms` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Failure Average Response Time by Flow | `ms` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Success Average Request Count by Endpoint | `short` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Success Average Request Count by Flow | `short` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Success Average Response Time by Endpoint | `ms` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
+| Outbound - Success Average Response Time by Flow | `ms` | 3.x-AM, 3.9.0, 3.9.1, 3.9.2
```

Comment:
I think we can say just say 3.9.x if we want (3.9.0, 3.9.1, 3.9.2 are the only ones available, so its the same)

Reply:
Okay!